app/views/videos/show/_translate_frame.html.erb

```diff
@@ -31,18 +31,21 @@
             <% end %>
           </div>
           <div role="tabpanel" class="tab-pane fade tab-srt" id="tab-srt">
+            <span>If you want to translate offline using an SRT, you can
             <% if (srt_link = video.amara_en_srt) %>
```

Comment:
If there is no English SRT available, can you remove "If you want to translate offline using an SRT, you can use the video above."? Flow seems a bit strange.